addons/point_of_sale/models/pos_config.py

```diff
@@ -691,8 +691,11 @@ def get_limited_products_loading(self, fields):
         """
         self.env.cr.execute(query, params + [20000])
         product_ids = self.env.cr.fetchall()
-        products = self.env['product.product'].search_read([('id', 'in', product_ids)], fields=fields)
-        return products
+        products = self.env['product.product'].search([('id', 'in', product_ids)])
+        product_combo = products.filtered(lambda p: p['detailed_type'] == 'combo')
+        product_in_combo = product_combo.combo_ids.combo_line_ids.product_id
+        products_available = products + product_in_combo
+        return products_available.read(fields)
```

Comment:
this implies that combo products are not always loaded. Is this the case? Otherwise there's no need for this code

Reply:
Is the case yes

Reply:
how can a combo product not be available in pos? I mean: what are the steps to arrive in that situation. Ideally it wouldn't be possible.

Reply:
we should make sure that all combo products from a combo that has `available_in_pos` set to `True` also have `available_in_pos` set to `True`

Reply:
When there are restricted categories in the PoS and a combo authorized to be loaded contains a product that is not.

Reply:
Need to ask a PO to know what to do in this situation

Reply:
ok, this makes sense, thanks!